src/box/txn.h

```diff
@@ -1038,6 +1038,20 @@ box_txn_savepoint(void);
 API_EXPORT int
 box_txn_rollback_to_savepoint(box_txn_savepoint_t *savepoint);
 
+/**
+ * Method to get txn's region to pass it outside the transaction manager.
+ * Do not use txn->region, use this method to track txn's allocations!
+ * Return region to txn with tx_region_release.
+ */
+struct region *
+tx_region_acquire(struct txn *txn);
+
+/**
+ * Method to return region to txn and account new allocations.
+ */
+void
+tx_region_release(struct txn *txn, enum tx_alloc_type alloc_type);
+
```

Comment:
Nit: Please move it before the box API function declarations.

Reply:
Done.